dependencies/precise/hammer_cli_foreman_discovery/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-hammer-cli-foreman-discovery (0.0.3-1) stable; urgency=low
+
+  * Update to 0.0.3
+
+ -- Ori Rabin <orabin@redhat.com>  The, 14 Apr 2016 12:52:53 +0200
```

Comment:
"Thu", better to use the scripts or standard deb tooling if you can, as these have to be correct.  (`scripts/update_package.rb -n hammer_cli_foreman_discovery -v 0.0.3`).